app/src/main/java/com/example/tudeeapp/presentation/screen/categoryDetails/state/CategoryDetailsUiState.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.tudeeapp.presentation.screen.categoryDetails.state
+
+import com.example.tudeeapp.domain.models.Category
+import com.example.tudeeapp.domain.models.Task
+
+data class CategoryDetailsUiState(
+    val isLoading: Boolean = true,
+    val errorMessage: String = "",
+    val tasks: List<Task> = emptyList(),
```

Comment:
We are dealing with the UI state not with the entity of domain layer
So use Task UI state instead, also for category